src/dsp_tools/resources/schema/data.xsd

```diff
@@ -73,7 +73,7 @@
         <xs:attribute name="encoding" type="encoding_types" use="required"/>
         <xs:attribute name="resrefs" type="resrefs_type"/>
         <xs:attribute name="comment" type="xs:string"/>
-        <xs:attribute name="permissions" type="xs:NCName" use="optional"/>
+        <xs:attribute name="permissions" type="xs:IDREF" use="optional"/>
```

Comment:
The "xs:IDREF" only works partially for me: I get warnings in VS Code, but the validation succeeds.

Reply:
Do you men the validation with dsp-tools? Because that is due to the issue we discussed on monday.